tools/retis_in_container.sh

```diff
@@ -20,6 +20,11 @@ else
 	exit -1
 fi
 
+# We can use a pseudo-terminal when using the pcap command (see 9f3361ac39c3),
```

Comment:
I think this could be improved a bit. As it is is not super clear. Can you please quickly revisit it?